components/notebook-controller/config/odh-metadata.yaml

```diff
@@ -0,0 +1,4 @@
+releases: 
```

Comment:
The Jira issue mentions the file name should be:
- `odh_metadata.yaml`

(notice the underscore)

Reply:
As shared in JIRA and Slack: https://redhat-internal.slack.com/archives/C060A5FJEAD/p1732213663369219
seems like the file name should be `component_metadata.yaml`
can we please make the change

Reply:
The JIRA we have open for our team lists: `odh_metadata.yaml` fwiw - I'll change that (as that was the source of truth I was going off of in my comment above)

Reply:
Hi @andyatmiami After a discussion in slack channel, the filename is decided to be component_metadata.yaml. I did add a comment in Epic [here](https://issues.redhat.com/browse/RHOAIENG-12521?focusedId=26066749&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-26066749)

Reply:
I updated the internal tracker for our team to now reflect this proper name so there is not any confusion.  Thanks for clarification/confirmation 💯